src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+function handleSuccess() {
+  const ressolveDiv = document.createElement('div');
+
+  ressolveDiv.className = 'message';
```

Comment:
Ensure that the variable name `ressolveDiv` is corrected to `resolveDiv` here as well to avoid reference errors.